### PR TITLE
Around you

### PR DESCRIPTION
**Pull request by RubenMattan**

### CODE REVIEW

**Comment by lucasvanhalst** [comment]
```diff
- Based on your location you will receive news
+ You will receive news based on your location

- We need your address to be able to receive location based news.\n\nUse the plus button to setup one or more addresses, such that you always receive the latest news.
sounds like we will receive news instead of the user
+ We need your address to be able to sent you news based on your location. ....

- The distance to where I wish to receive news is ...
This sounds a bit odd, not sure how to properly word it though
```

File: rogerthat/src/main/java/com/mobicage/rogerthat/plugins/system/ProfileAddAddressActivity.java [hunk 1]
```diff
@@ -377,4 +451,47 @@ private SafeBroadcastReceiver getBroadcastReceiver() {
...
+            mCurrentCircle.remove();
+        }
+
+        mCurrentCircle = mMap.addCircle(new CircleOptions()
+                .center(mCenter)
+                .radius(mDistance)
+                .strokeWidth(0f)
+                .fillColor(0x550000FF));
+
+        if (mDistance <= 500) {
```
Comment (lucasvanhalst) [inline rogerthat/src/main/java/com/mobicage/rogerthat/plugins/system/ProfileAddAddressActivity.java]: More accurate zoom:
```java
LatLngBounds bounds = new LatLngBounds.Builder()
                .include(SphericalUtil.computeOffset(mCenter, mDistance, 0))
                .include(SphericalUtil.computeOffset(mCenter, mDistance, 90))
                .include(SphericalUtil.computeOffset(mCenter, mDistance, 180))
                .include(SphericalUtil.computeOffset(mCenter, mDistance, 270))
                .build();
mMap.animateCamera(CameraUpdateFactory.newLatLngBounds(bounds, 100);
```

File: rogerthat/src/main/java/com/mobicage/rogerthat/plugins/system/ProfileAddAddressActivity.java [hunk 2]
```diff
@@ -313,62 +504,68 @@ private void saveAddress() {
...
+                        final String houseNumberText = mHouseNumber.getText().toString().trim();
+
+                        mCallbackKey = UUID.randomUUID().toString();
+                        if (mOldAddress == null) {
+                            mSystemPlugin.addAddress(mCallbackKey, labelText, mCurrentStreet, houseNumberText, busNumberText,
+                                    mCurrentZipcode, mCurrentCity, countryCode, mDistance, (float) a.getLatitude(), (float) a.getLongitude());
+                        } else {
+
+                        }mSystemPlugin.updateAddress(mCallbackKey, mOldAddress.uid, labelText, mCurrentStreet, houseNumberText, busNumberText,
+                                mCurrentZipcode, mCurrentCity, countryCode, mDistance, (float) a.getLatitude(), (float) a.getLongitude());
```
Comment (bart-at-mobicage) [inline rogerthat/src/main/java/com/mobicage/rogerthat/plugins/system/ProfileAddAddressActivity.java]: oops, this line is not inside the else block

**Comment by RubenMattan** [comment]
> ```diff
> ased on your location you will receive ne
> ```

@lucasvanhalst  Copied from iOS Talk to Maureen or Bart if this need to be changed

**Pull request merged**